fe/src/pages/Signup.js

```diff
@@ -0,0 +1,92 @@
+import { useState } from "react";
+
+export default function Signup() {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+  const [age, setAge] = useState(null);
+  const [height, setHeight] = useState(null);
+  const [weight, setWeight] = useState(null);
+  const [isMale, setIsMale] = useState(true);
+
+  const handleChangeUsername = (e) => {
+    setUsername(e.target.value);
+  };
+
+  const handleChangePassword = (e) => {
+    setPassword(e.target.value);
+  };
+
+  const handleChangeAge = (e) => {
+    setAge(parseInt(e.target.value));
+  };
+
+  const handleChangeAge = (e) => {
+    setAge(parseInt(e.target.value));
+  };
+
+  const handleChangeHeight = (e) => {
+    setHeight(parseInt(e.target.value));
+  };
+
+  const handleChangeWeight = (e) => {
+    setWeight(parseInt(e.target.value));
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    console.log(email);
+    console.log(password);
+  };
+
+  return (
+    <div>
+      <form onSubmit={handleSubmit}>
+        <div>
+          <label for="username">Username</label>
+          <input
+            type="username"
+            value={username}
+            onChange={handleChangeUsername}
+          ></input>
+        </div>
+        <div>
+          <label for="password">Password</label>
+          <input
```

Comment:
Hey Ji the components can be further refractory inside Signup component. But if you get time you can do that later.

Reply:
There are conflicts in package-lock.json MIT license which is added for modules. I am trying to figure that out too.